client/components/jetpack/licensing-activation-banner/style.scss

```diff
@@ -0,0 +1,45 @@
+.licensing-activation-banner {
+	padding-top: 16px;
+    padding-bottom: 16px;
+    color: var( --studio-white );
+    text-align: center;
+
+	border-width: 1px 0;
+	border-style: solid;
+	border-color: var( --studio-black );
+	background-color: var( --studio-black );
+
+	@include breakpoint-deprecated( '>660px' ) {
+        display: flex;
+        justify-content: center;
+        align-items: center;
```

Comment:
I think we can apply these properties even for devices `<660px` (i.e. move it outside the media query), with one small addition: `flex-wrap: wrap` (it will cause the flex items to wrap once they become too wide for the container). 

Before | After
--- | ---
![Screen Capture on 2022-01-11 at 10-50-57](https://user-images.githubusercontent.com/1749918/148913596-44945e69-acfb-48ba-b8c6-bb9eb9ca2c86.gif) | ![Screen Capture on 2022-01-11 at 10-53-36](https://user-images.githubusercontent.com/1749918/148913662-461179ab-aceb-44b1-a005-d3efdd205262.gif)

Reply:
This is a nice suggestion.